tests/static_libs.rs

```diff
@@ -46,7 +46,16 @@ fn expected_linker_parts() -> &'static [&'static str] {
     }
     #[cfg(target_os = "macos")]
     {
-        &["-liconv", "-lSystem", "-lc", "-lm"]
+        &[
+            "-framework",
+            "Security",
+            "-framework",
+            "CoreFoundation",
+            "-liconv",
+            "-lSystem",
+            "-lc",
+            "-lm",
+        ]
     }
     #[cfg(target_os = "windows")]
     {
```

Comment:
I think our `static_libs.rs` test isn't being run on Windows anymore since I removed `Makefile.windows` in https://github.com/rustls/rustls-ffi/pull/396. Previously we ran the test there, and I had updated the CI for the Windows CMake builds to only run the client/server integration test. I forgot to update that when removing the makefile :sob: I will look into fixing this separately.

In an ideal world this test would have failed for this branch, complaining about the expected list missing `crypt32.lib` and pointing to that as the right fix for the Windows CI issue you observe.

Reply:
Thanks for looking into this, I cherry-picked your 575c80dd4e2e5841ea952d71f6dee1ff9c9aeb15 commit for this PR :+1: